### PR TITLE
ARROW-1994: [Python] Test against Pandas master

### PR DESCRIPTION
**Pull request by kszucs**
Add to crossbow tests.yml as well after merging https://github.com/apache/arrow/pull/2755

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -181,6 +181,21 @@ services:
       context: .
       dockerfile: integration/hdfs/Dockerfile
 
+  pandas-master:
+    # Usage:
+    #   export PYTHON_VERSION=3.6
+    #   docker-compose build cpp
+    #   docker-compose build python
+    #   docker-compose build pandas-master
```
Comment (xhochy) [inline docker-compose.yml]: Can you add the necessary command line options to do a no-cache-rebuild of the pandas-master image to get the real latest pandas commit, not the one that was cached by docker?

Comment (kszucs) [reply docker-compose.yml]: Sure, good point!

**Review by xhochy — APPROVED** [review]
+1, LGTM

Does it actually work?

**Comment by kszucs** [comment]
I didn't rebuild the image, but plasma tests and a pandas serialization test are failing. I'll add this as a nightly as well.

**Comment by kszucs** [comment]
Created ticket for fixing plasma errors https://issues.apache.org/jira/browse/ARROW-3709

**Pull request closed without merging**